platform/security/servlet/security-servlet-logout/src/main/java/org/codice/ddf/security/servlet/logout/LogoutService.java

```diff
@@ -80,17 +80,14 @@ public Response getActionProviders(@Context HttpServletRequest request)
                             "",
                             true);
 
-                    if (displayName != null
-                            && !displayName.equals(SubjectUtils.GUEST_DISPLAY_NAME)) {
-                        actionProperties.put("title", action.getTitle());
-                        actionProperties.put("realm", realm);
-                        actionProperties.put("auth", displayName);
-                        actionProperties.put("description", action.getDescription());
-                        actionProperties.put("url",
-                                action.getUrl()
-                                        .toString());
-                        realmToPropMaps.add(actionProperties);
-                    }
+                    actionProperties.put("title", action.getTitle());
```

Comment:
I think that we should test that `realmToPropMaps` always has values added to it no matter what the value of the `displayName` is.